Dockerfile

```diff
@@ -26,11 +26,14 @@ RUN apt-get update && apt-get install -y --no-install-recommends gconf-service l
     --no-install-recommends \
   && npm install -g yarn@$YARN_VERSION \
   && npm install -g s3-cli \
-  && npm install -g codeclimate-test-reporter \
```

Comment:
the npm codeclimate-test-reporter was deprecated. https://www.npmjs.com/package/codeclimate-test-reporter